.circleci/config.yml

```diff
@@ -306,60 +306,54 @@ jobs:
     docker:
       - image: circleci/php:7.4.15-apache-browsers
     steps:
+      - hosts_keyscan
       - checkout_code
-      - run:
-          name: Checkout edge branch
-          command: |
-            cd ~/project
-            git checkout -b $EDGE_BUILD_BRANCH
       - install_php_os_extensions
-      - install_platformsh_tools
+      - install_psh_cli
       - run:
-          name: Trigger a data sync from production environment to nightly edge build.
```

Comment:
Dropped the nightly references as the cron syntax indicates frequency. Sticking to just 'edge' as a way of designating the edge build/environment.